### DISCUSSION TITLE
Site doesn't work with Ghostery

### DISCUSSION THREAD

**Discussion by KoalaBear84** [quasarframework/quasar#14462] [General - Components / Directives / etc] [answered]
Some things don't work in the website when Ghostery is enabled. Especially:
https://quasar.dev/start/umd

https://chrome.google.com/webstore/detail/ghostery-%E2%80%93-privacy-ad-blo/mlomiejdfkolichcflejclcbmpeaniij

**Comment by MilosPaunovic** [comment]
Which particular things don't work, you need to be more specific, is it the entire UMD section or...?

**Comment by KoalaBear84** [comment]
The part that doesn't work is the sidebars, they don't show up, and clicking any of the switches doesn't work.

It can be tested when you clear all the cache / cookies.

Message in console:
GET https://quasar.dev/assets/gdpr.js net::ERR_BLOCKED_BY_CLIENT

**Comment by yusufkandemir** [comment, marked as answer]
This has been addressed in https://github.com/quasarframework/quasar/commit/1a3019d54dc79a58d69f8c7ad8445d4f3eae9260.
The docs have been deployed like 5 hours ago or less than that. Please try again, but make sure you get the latest version before trying.

**Comment by KoalaBear84** [comment]
Yes, it's working now, also when the script cannot be loaded 👍
